suitcase/iss_series.py

```diff
@@ -0,0 +1,84 @@
+# ingest Series files into databroker (for Materials Project)
+import uuid
+import time
+import numpy as np
+
+from databroker import Broker, temp_config
+
+
+def parse_series_file(fname, prefix=None, **kwargs):
+    if prefix is None:
+        prefix = ""
+
+    res = np.loadtxt(fname, **kwargs).T
+    energies = res[0]
+    data = res[1:]
+    data_dict = dict()
+
+    for i in range(len(data)):
+        key = prefix + f"{i}"
+        data_dict[key] = data[i]
+
+    return energies, data_dict
+
+
+def ingest(fname, headerprefix=None, md=None):
```

Comment:
👍 
we insert those docs into analysis database, right? This is like prototype of analysis store.